basics/undo-installation.md

```diff
@@ -0,0 +1,36 @@
+# Undo installation
+
+Nix supports atomic *rollback* of your packages to a previous state. *Rollback* is the same as undoing your last action (or actions).
+
```

Comment:
Maybe a quick explanation of what is a generation in the paragraph above. I guess it's like the git reflog for environment change? Could use a glossary definition as well